### PR TITLE
MM-17425 Disable Send button when sending post (1.22)

### PR DESCRIPTION
**Pull request by hmhealey**
This is a 1.22-specific version of the fix for MM-17425 contained in https://github.com/mattermost/mattermost-mobile/pull/3099

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-17425

#### Checklist
- Includes text changes and localization file updates

#### Device Information
This PR was tested on: iOS Simulator (iPhone X, iOS 12.1), Android Emulator (Nexus 5X, Android 9)

### CODE REVIEW

**Comment by enahum** [comment]
@hmhealey build fails

File: assets/base/i18n/en.json [hunk 1]
```diff
@@ -351,9 +351,6 @@
   "mobile.post_pre_header.flagged": "Flagged",
   "mobile.post_pre_header.pinned": "Pinned",
   "mobile.post_pre_header.pinned_flagged": "Pinned and Flagged",
-  "mobile.post_textbox.empty.message": "You are trying to send an empty message.\nPlease make sure you have a message or at least one attached file.",
```
Comment (enahum) [inline assets/base/i18n/en.json]: why are these removed?

**Comment by enahum** [comment]
@hmhealey build failing

**Comment by lindalumitchell** [comment]
QA can test this one after it's merged rather than on the PR. Removing QA Review label.

File: assets/base/i18n/en.json (hunk 1 above)
Comment (hmhealey) [reply assets/base/i18n/en.json]: It was actually impossible to trigger that modal. We check `this.canSend()` at the start of `handleSendMessage` which does an early return any time that modal would've been displayed.
